include/MainWindow.h

```diff
@@ -55,9 +56,9 @@ class MainWindow : public QMainWindow
 {
 	Q_OBJECT
 public:
-	QMdiArea* workspace()
+	inline QMdiArea* workspace()
```

Comment:
The `inline` keyword isn't needed. Member functions defined in the header are automatically `inline`.